ros_ws/src/auto_race_pg/include/remote_keyboard.h

```diff
@@ -0,0 +1,56 @@
+#ifndef REMOTE_KEYBOARD_H
+#define REMOTE_KEYBOARD_H
+
+#include <ros/ros.h>
+
+#include <std_msgs/Float64.h>
+#include <std_msgs/Int64.h>
+
+#include <signal.h>
+#include <termios.h>
+#include <time.h>
+
+#define MODE "keyboard"
+
+#define TOPIC_SPEED "/set/speed"
+#define TOPIC_ANGLE "/set/position"
+
+#define TOPIC_STATUS_DMS "/status/dms"
+
+#define KEYCODE_W 119
+#define KEYCODE_A 97
+#define KEYCODE_S 115
+#define KEYCODE_D 100
+#define KEYCODE_SPACE 32
+
+#define DELTA_SPEED_UP 0.2
+#define DELTA_SPEED_DOWN 0.1
```

Comment:
I understand what DELTA_SPEED_UP is supposed to do (all changes in speed are in increments of DELTA_SPEED_UP) but what is the meaning of DELTA_SPEED_DOWN?

Reply:
if "W" is not pressed, the speed should go down by DELTA_SPEED_DOWN

Reply:
has been removed all together